### DISCUSSION TITLE
I would like to uninstall the program and reset all settings.

### DISCUSSION THREAD

**Discussion by jujuPARIS** [linearmouse/linearmouse#193] [General]
Hi ;) Everything is in the title, how to uninstall completely the program and settings ? 

thanks

**Comment by lujjjh** [comment]
1. Quit LinearMouse.
2. Move LinearMouse.app to Trash.
3. *(Optional)* Open Finder, press <kbd>⇧ ⌘ G</kbd>, go to `~/.config` and remove the `linearmouse` folder.
